client/src/components/Image/index.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import Modal from 'react-bootstrap/Modal';
```

Comment:
Please add `react-bootstrap/Modal` in package.json as well. Otherwise, the build is failing.